components/raftstore/src/store/msg.rs

```diff
@@ -741,7 +741,7 @@ pub enum PeerMsg<EK: KvEngine> {
     /// Raft message is the message sent between raft nodes in the same
     /// raft group. Messages need to be redirected to raftstore if target
     /// peer doesn't exist.
-    RaftMessage(InspectedRaftMessage),
+    RaftMessage(InspectedRaftMessage, Option<Instant>),
```

Comment:
Ditto. How about add one extra metering metric `send_time` into `InspectedRaftMessage`.

Reply:
I think the send_time is only interested when InspectedRaftMessage is inside the Peer::RaftMessage.